src/field.rs

```diff
@@ -199,6 +181,26 @@ pub trait FieldElement:
     }
 }
 
```

Comment:
True, I had just moved those two because Poplar wouldn't need them (it just squares), so "FFT friendly field element" is a slight misnomer because of that. I could implement those two for a cleaner API boundary.

Reply:
For now, I would not have `Field255` implement them. Rather, I would just stick a `todo!("implement when needed")` in there.